src/components/activity/ActivityDecomposition.svelte

```diff
@@ -59,8 +59,9 @@
         <TreeLeafIcon />
       {/if}
     </button>
-    <!-- svelte-ignore a11y-click-events-have-key-events -->
-    <span on:click={() => dispatch('select', rootSpanId)} on:dblclick={toggle} class={nodeClass}>{type}</span>
+    <span role="none" on:click={() => dispatch('select', rootSpanId)} on:dblclick={toggle} class={nodeClass}>
```

Comment:
In the future we might want to consider using actual buttons instead of divs with "role=none" for accessibility reasons?

Reply:
Agree! Created https://github.com/NASA-AMMOS/aerie-ui/issues/747